src/content/ui/adaptive-responsive/idioms.md

```diff
@@ -0,0 +1,398 @@
+---
+title: Platform idioms
+description: >-
+  Learn how to create a responsive app >
+  that responds to changes in the screen size. 
+short-title: Idioms
+---
+
+<?code-excerpt path-base="ui/adaptive_app_demos"?>
+
+{% comment %}
+<b>PENDING: Leave this page out for now... In V2, I'd like to include it. Mariam suggested splitting it up by platform and I like that idea</b>
```

Comment:
Right now this page will still be discoverable. Do you want it to not be published? Not indexed? Etc. Or is that fine?